test/client_test.rb

```diff
@@ -68,6 +68,11 @@ def test_ping
     assert_equal 'Scope All is OK', client.ping(scope: 'all')
   end
 
+  def test_query
+    assert (client.query { paginate(Ref.new('classes')) })[:data].is_a?(Array)
+    assert client.query(Fauna.query { paginate(Ref.new('classes')) })[:data].is_a?(Array)
```

Comment:
I'd like to see the non `Fauna.query` form tested as well, and that the output of all three are the same.